### ISSUE TITLE
Improvement of "comment" notifications.

### ISSUE THREAD

**Issue by TheCoward** [Retrospring/bugs#10]
I thought it would be nice if users had the possibility to have a few more notifications.
Currently, there notifications for answers, smiles on your answers and comments on your answers.
![notifications](https://a.pomf.se/wjrmue.png)
What if you implemented the possibility to get notified on (comments on) answers to *your* questions, and not only your own answers, or the possibility to get notified on comments to answers you smiled, get notified on new comments if you have commented on an answer, and finally, add a Facebook-like grouping of notifications such as "X and X commented on your answer" to reduce the quantity of notifications? All of which could be enabled or not, to the user's liking.

**Comment by yukimono** [comment]
Easily done with a ``subscriptions`` table, every time someone comments on the post, it just notifies everyone that is subscribed to it that ISN'T the commenter.

```ruby
Subscription.where("target_id = ? AND user_id != ?", answer.id, commenter.id).each do |subscription|
  Notificaiton.new(target_type: "Comment", target_id: comment.id, recipient_id: subscription.user_id)
end
```

I'll probably implement this and push another pull request sometime in the next few days.

**Comment by TheCoward** [comment]
Yes, that was the first idea I thought of, but didn't write in my post: simply a subscribe button like well... right here.
![github](https://a.pomf.se/jfdebr.png)

**Comment by yukimono** [comment]
Reopening due to reworking notification slugs
